src/main/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/RoleAssignmentRequestResource.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.ccd.data.casedataaccesscontrol;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Builder
+@Data
+@NoArgsConstructor
```

Comment:
FYI - there is a new annotation available in lombok @Jacksonized. You can avoid NoArgsConstructor altogether with `@Value and @Jacksonized`

Reply:
Resolved in commit 96adc047a20c6d8d70f4b91d4d9b72e58c62b30e

Sorry, I forgot to perform my usual step of simplifying my Lombok annotations on my POJOs after running tests with coverage.  Thanks for the `@Jacksonized` tip, I have bumped Lombok and updated my POJOs.